services/web-app/components/video/video.module.scss

```diff
@@ -0,0 +1,73 @@
+@use '@carbon/react/scss/motion' as motion;
+@use '@carbon/react/scss/colors' as colors;
+@use '@carbon/react/scss/theme' as theme;
+@use '@carbon/react/scss/spacing' as spacing;
+
+.video {
+  width: 100%;
+}
+
+.video-container {
+  position: relative;
+  display: table;
+  width: 100%;
+}
+
+div.video-button {
+  position: absolute;
+  z-index: 10;
+  top: 0%;
+  right: 0%;
+  bottom: 0%;
+  left: 0%;
+  display: flex;
+  width: 4.5rem;
+  height: 4.5rem;
+  align-items: center;
+  justify-content: center;
+  margin: auto;
+  background-color: colors.$blue-60;
+  border-radius: 50%;
+  transition: all motion.$duration-moderate-01 motion(entrance, productive);
+}
+
+div.video-button:focus {
+  outline: 2px solid theme.$focus;
+  outline-offset: -2px;
+}
+
+div.video-button:hover {
+  background-color: colors.$gray-90;
+  opacity: 0.8;
+}
+
+div.video-button > svg {
+  margin-left: spacing.$spacing-02;
+  fill: theme.$icon-inverse;
+  transition: all motion.$duration-moderate-01 motion(entrance, productive);
+}
+
+div.video--is-playing > svg {
+  margin-left: 0;
+}
+
+// iframe
+.vimeo {
+  position: relative;
+  overflow: hidden;
+  height: 0;
+  padding-bottom: 56.543%;
```

Comment:
this number feels weird (maybe just me..hehe) is there a better way to do this?

Reply:
Oooh, this weirdness is a trick to get the vimeo iframe to be responsive I believe.

Reply:
yep! looks like a trick to get the video to assume full container width, see: https://stackoverflow.com/questions/42858199/vimeo-full-width
I added a comment in the scss file